src/kit.ts

```diff
@@ -346,8 +346,12 @@ export async function vsInstallations(): Promise<VSInstallation[]> {
     return [];
   }
 
-  const vs_installs = JSON.parse(vswhere_res.stdout) as VSInstallation[];
-  for (const inst of vs_installs) {
+  let vs_installs = JSON.parse(vswhere_res.stdout) as VSInstallation[];
+  for (let inst of vs_installs) {
+    inst.majorVersion = parseInt(inst.installationVersion)
```

Comment:
`majorVersion` is not an attribute on `VSInstallation`, so this won't work. Instead, make this a local variable `majorVersion` for doing the interpolation on line 353.

Reply:
OK thanks.